automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/DownloadFixtures.scala

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.dsde.firecloud.fixture
+
+import com.typesafe.scalalogging.LazyLogging
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.attribute.PosixFilePermission
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import org.broadinstitute.dsde.firecloud.component.Clickable
+import org.broadinstitute.dsde.workbench.service.util.Util
+import org.scalatest.concurrent.Eventually
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.service.test.WebBrowserUtil
+import org.openqa.selenium.WebDriver
+
+trait DownloadFixtures extends Eventually with LazyLogging with WebBrowserUtil {
+
+
+  def makeTempDownloadDirectory(): String = {
+    /*
+     * This might work some day if docker permissions get straightened out... or it might not be
+     * needed. For now, we instead `chmod 777` the directory in run-tests.sh.
+    new File("chrome").mkdirs()
+    val downloadPath = Files.createTempDirectory(Paths.get("chrome"), "downloads")
+    val permissions = Set(PosixFilePermission.OWNER_WRITE, PosixFilePermission.GROUP_WRITE, PosixFilePermission.OTHERS_WRITE)
+    Files.setPosixFilePermissions(downloadPath, permissions.asJava)
+    downloadPath.toString
+     */
+
+    val downloadPath = s"chrome/downloads/${UUID.randomUUID()}"
+    val dir = new File(downloadPath)
+    dir.deleteOnExit()
+    dir.mkdirs()
+    val path = dir.toPath
+    logger.info(s"mkdir: $path")
+    val permissions = Set(
+      PosixFilePermission.OWNER_WRITE, PosixFilePermission.OWNER_READ, PosixFilePermission.OWNER_EXECUTE,
+      PosixFilePermission.GROUP_WRITE, PosixFilePermission.GROUP_READ, PosixFilePermission.GROUP_EXECUTE,
+      PosixFilePermission.OTHERS_WRITE, PosixFilePermission.OTHERS_READ, PosixFilePermission.OTHERS_EXECUTE)
+    import scala.collection.JavaConverters._
+    Files.setPosixFilePermissions(path, permissions.asJava)
+    path.toString
+  }
+
+  /**
+    * Downloads the metadata currently being viewed.
+    *
+    * If downloadPath is given, the file is given a timestamped name and moved from that location
+    * into the "downloads" directory off the current working directory. This serves two purposes:
+    *
+    * 1. Archiving the file for later inspection when tests fail
+    * 2. Keeping the browser download directory clean so that it doesn't auto-rename subsequent
+    * downloads with the same filename
+    *
+    * @param downloadPath the directory where the browser saves downloaded files
+    * @return the relative path to the moved download file, or None if downloadPath was not given
```

Comment:
Remove "or None if downloadPath was not given" now that `downloadPath` is not an `Option`. (I don't remember why it used to be an `Option`.)